### PR TITLE
add me

### PR DESCRIPTION
**Pull request by sylv256**
Long have we awaited,
Tehc activated.

### CODE REVIEW

**Review by FoundationGames — APPROVED** [review]

**Comment by sylv256** [comment]
im a very popular cringe singularity and everyone is always thinking about me /s

File: data/person [hunk 1]
```diff
@@ -84,3 +84,4 @@ Carter
 Acrogenous
 Whangd00dle
 Will BL
+Tehc
```
Comment (Platymemo) [inline data/person:87]: Is this really necessary? I think this PR would be great without this one line

**Review by Platymemo — CHANGES_REQUESTED** [review]
Almost good, one slight problem tho.

File: data/person (hunk 1 above)
Comment (sylv256) [reply data/person:87]: get out.

**Review by sylv256 — COMMENTED** [review]
I am the most based human on planet earth

**Comment by sylv256** [comment]
I can tell this is a very controversial pr, almost as much as that one heated fluid api pr that kept getting closed

**Comment by FoundationGames** [comment]
🚲🏚

**Review by IMS212 — APPROVED** [review]
very based

**Comment by sylv256** [comment]
if this doesn't get merged in 8 hours I'm pinging @everyone

**Comment by sylv256** [comment]
> if this doesn't get merged in 8 hours I'm pinging @everyone 

why is someone's username "everyone"

**Review by valoeghese — APPROVED** [review]
seems fine tbh

**Review by williambl — APPROVED** [review]
Don't know how this was missing for so long

**Review by Su5eD — APPROVED** [review]
There is no option to deny, I can only approve :(

**Review by lucsoft — COMMENTED** [review]
Looks good but i think there is a typo you mean TehcJS right?

**Comment by valoeghese** [comment]
he is known as Tehc on discord

**Pull request merged**